Civi/Funding/DocumentRender/CiviOffice/CiviOfficeDocumentRenderer.php

```diff
@@ -20,33 +20,17 @@
 namespace Civi\Funding\DocumentRender\CiviOffice;
 
 use Civi\Funding\DocumentRender\DocumentRendererInterface;
-use Civi\RemoteTools\Api3\Api3Interface;
-use Webmozart\Assert\Assert;
 
 class CiviOfficeDocumentRenderer implements DocumentRendererInterface {
 
-  private Api3Interface $api3;
+  private CiviOfficeRenderer $renderer;
```

Comment:
What's the point having this basically forward every method to a new class?

Reply:
The `render()` method is not a pure forward. So the behavior of the class didn't change and the new class `CiviOfficeRenderer` can be used in `ApplicationTemplateRenderController`, too.